src/templates/dotnet/parseDotnetTemplates.ts

```diff
@@ -74,6 +74,7 @@ function parseDotnetTemplate(rawTemplate: IRawTemplate): IFunctionTemplate {
     return {
         isHttpTrigger: /^http/i.test(rawTemplate.Name) || /webhook$/i.test(rawTemplate.Name),
         isTimerTrigger: /^timer/i.test(rawTemplate.Name),
+        isSqlBindingTrigger: /Sql.*Binding/i.test(rawTemplate.Name),
```

Comment:
`/Sql.*Binding/i` should maybe be a constant since it's used here and in `parseScriptTemplates.ts`.